site/config.toml

```diff
@@ -32,7 +32,7 @@ pygmentsStyle                 = "manni"
 
 # SEO
 [params]
-    description               = "We're a community of Latinx professionals in the tech industry. 🌮️"
+    description               = "We're a community of Latinx professionals in the tech industry. 🌮️ This space is for all Latinxs, and that includes the regions of the Caribbean, Haiti and Brazil as well as Afro-Latinx and Indigenous, Queer & Trans communities. Latinx is used instead of Latino or Latinos because it is a gender-neutral and inclusive term"
```

Comment:
Could we get a period added at the end though? 😛 

Also, I'm not sure indigenous, queer & trans need to be capitalized. Also, we could probably just use LGBTQ instead which is more widely understood and encompasses all the identities.

https://www.glaad.org/reference/transgender

> We're a community of Latinx professionals in the tech industry. 🌮️ 

> This space aims to be inclusive so we invite Latinx from the regions of the Caribbean, Haiti, and Brazil as well as those who identify as Afro-Latinx or LGBTQ.

> The term "Latinx" is used instead of Latino because it is a gender-neutral and inclusive term.

Reply:
What do you think of the readjusted text @fvntr? I can change in a new PR.